spec/models/master_file_spec.rb

```diff
@@ -563,8 +563,12 @@ class MyEncoder < ActiveEncode::Base
 
   describe "waveform generation" do
     subject(:master_file) { FactoryBot.create(:master_file) }
+    let(:collection) { FactoryBot.create(:collection, name: 'rails env waveform test') }
+    let(:media_object) { MediaObject.new }
 
     it 'runs the waveform job' do
+      media_object.collection = collection
+      master_file.media_object = media_object
       expect(WaveformJob).to receive(:perform_now).with(master_file.id)
```

Comment:
Fixes a test failure depending the config setting, SETTINGS__MASTER_FILE_MANAGEMENT__STRATEGY, that moves file uploaded via the UI into a useful location/name and not leave in the `upload` directory with the potential to be overwritten with a file of the same name https://github.com/ualbertalib/avalon/pull/454